bindings/rust/wasmedge-sys/src/module.rs

```diff
@@ -30,4 +31,36 @@ impl Module {
 
         Ok(Self { ctx })
     }
+
+    pub fn load_from_buffer(
+        config: &crate::config::Config,
+        buffer: Vec<u8>,
+    ) -> Result<Self, ErrReport> {
+        let loader_ctx = unsafe { wasmedge::WasmEdge_LoaderCreate(config.ctx) };
+        let mut ctx: *mut wasmedge::WasmEdge_ASTModuleContext = std::ptr::null_mut();
+
+        assert!(!buffer.is_empty(), "WasmEdge fail to load an empty buffer");
+
+        let ptr = unsafe {
+            let ptr = libc::malloc(buffer.len());
```

Comment:
Do we need to free this memory before exiting this function?

Reply:
Yes, we need to delete it, and there is a test to make sure this buffer can be deleted, and the VM still works.  
Thanks.